csrc/scheduler/normalization.cpp

```diff
@@ -1154,19 +1165,18 @@ std::shared_ptr<ReductionParams> outerPersistentHeuristic(
 
   // Always need to mark inner reduction unroll for rfactor in outer persitent
   // kernels
-  rparams->unroll_factor_inner_reduction = inner_reduction_unroll_factor;
+  rparams->unroll_factor_inner_reduction = hp.redu_unroll_factor.get();
 
-  rparams->unroll_factor_iter_dom = iter_unroll_factor;
+  rparams->unroll_factor_iter_dom = hp.iter_unroll_factor.get();
```

Comment:
Isn't this always ignored?

https://github.com/NVIDIA/Fuser/pull/553/files#diff-94d037e9fcb9914df95a45c2b1e5966bb04f10842beebcdb08a17abaa8b62301R256

Reply:
We still have vectorize (unroll iter dom), although it is set to 1 in most cases of this PR.
```
    if (rparams.vectorize_iter_dom) {
      vectorize(iter_axis, rparams.unroll_factor_iter_dom);
    }

```
I did another test with a iter domain size of 3276800 and `iter_unroll_factor` will be set to 4 in this PR. Doing unswitch can improve the performance from 1030 GB/s to 1060 GB/s.
I reversed the changes to this file since  `iter_unroll_factor = 1` if `total_iteration_numel < 2097152` and the unswitch will not be used.